yellowbrick/utils.py

```diff
@@ -155,6 +157,45 @@ def is_dataframe(obj):
 isdataframe = is_dataframe
 
 
+#From here: http://stackoverflow.com/questions/26248654/numpy-return-0-with-divide-by-zero
+def numpy_div0( a, b ):
+    """
+    Ufunc-extension that returns 0 instead of nan when dividing numpy arrays
+    example: >> div0( [-1, 0, 1], 0 ) -> [0, 0, 0]
+    """
+    with np.errstate(divide='ignore', invalid='ignore'):
+        c = np.true_divide( a, b )
+        c[ ~ np.isfinite( c )] = 0  # -inf inf NaN
+    return c
+
+
+def find_text_color(base_color, text_colors=("black","white")):
```

Comment:
In terms of the signature of the function, why have you chosen to use a tuple of text_colors? Wouldn't the below function signature be just as simple, but perhaps a bit more explanatory (for example, if I wanted to make the light color blue but keep the dark color the default):

```
def find_text_color(base_color, dark_color="black", light_color="white"):
    pass
```

While we're at it, there is a semantic repetition of the `_color` that could probably be omitted since it's part of the function signature, but that's more stylistic than anything else. 

```
def find_text_color(background, dark_text="black", light_text="white"):
    pass
```